AlgebraInLean/Chapter01/Sheet03.lean

```diff
@@ -0,0 +1,91 @@
+import AlgebraInLean.Chapter01.Sheet02
+
+namespace AlgebraInLean
+
+variable {α : Type*}
+
+/-
+When we defined the monoids and groups, we only required the *existence* of identity and inverses,
+but not their uniqueness. However, it does happen that they must be unique, as shown below.
+-/
+
+/--
+Uniqueness of the identity element in a monoid. If any element e₂ "behaves like" the identity, then
+it must be equal to the identity.
+-/
+theorem id_unique [Monoid α] (e₂ : α) (h : ∀ (a : α), (μ a e₂ = a ∧ μ e₂ a = a)) : e₂ = 𝕖 := by
+  /-
+  The `obtain` tactic may be useful. Given a hypothesis `h : P ∧ Q`, `obtain ⟨h₁, h₂⟩ := h` creates
+  hypotheses `h₁ : P` and `h₂ : Q`. This also works to destructure other types like `∃`.
+  -/
+  -- sorry
+  -- SAMPLE SOLUTION
+  specialize h 𝕖
+  obtain ⟨h, _⟩ := h -- we don't care about the property on the right, so we use `_` instead.
+  rw [id_op] at h
+  exact h
+  -- END OF SAMPLE SOLUTION
+
+variable [Group α]
+
+/--
+Uniqueness of the inverse of an element. If any element i "behaves like" the inverse of a, then it
```

Comment:
add backticks around the `i`. Can look like a typo without, since it is a lowercase i.